content/guides/outliers.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Outlier Detection (BETA)
+kind: documentation
+sidebar:
+  nav:
+    - header: Outlier Detection
+    - text: Show Outliers in Dashboards
+      href: "#dashboards"
+    - text: Alert on Outliers
+      href: "#alerts"
+    - text: Algorithms & Parameters
+      href: "#algorithms"
+---
+
+Outlier Detection is a beta feature. It allows you to detect when some members of a group are behaving strangely compared to the others. For example, you could detect that one web server in a pool is processing an unusual number of requests, and hence should be a target for replacement. Or, you could get an early warning that significantly more 500s are happening in one AWS Availability Zone (AZ) than the others, which might indicate an issue brewing in that AZ.
+
+<img src="/static/images/outliers/outliers-metric-alert.png" style="width:100%; border:1px solid #777777"/>
+
+## How to Use Outlier Detection on Your Data
+
+Starting in the beta period, we’ve added a new query function called `outliers` to our query language. This function acts as a filter, selecting only the series in a group that are outliers compared to the rest of the group.
+
+You can use this function to display and alert on outliers in your data. To try it out, you’ll first need a metric for which a group of hosts (or availability zones, partitions, etc) should exhibit uniform behavior. For the function to work, be sure that there are at least 3 or more members in the group. Given that, here are two ways to use outlier detection on that group.
```

Comment:
I want "and" here to show that you can display (dashboards) and alert (monitors) on outliers